ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PauseResumeIntegrationTest.java

```diff
@@ -260,17 +251,18 @@ private int getRunningCount() {
     return getCount(KsqlQueryStatus.RUNNING);
   }
 
+  // JNH: Fix this.
```

Comment:
Out of curiosity, is this referring to the comment below that says `This failing shows that the command topic isn't being read completely?`? I'd be curious to understand more about what the issue is.

Reply:
Fixed it!